kensu/utils/kensu.py

```diff
@@ -335,6 +335,20 @@ def in_mem(self, var_name):
     def name_for_stats(self, input_name):
         return input_name.replace('.', '_')
 
+    def extract_stats(self, stats_df):
+        try:
+            stats = self.extractors.extract_stats(stats_df)
+        except Exception as e:
+            logging.info(f'stats extraction from {type(stats_df)} failed', e)
```

Comment:
Warn/error?

Reply:
IMHO error is too much (for automagic collector), but `warn` makes sense

@SammyEK ?

Reply:
would say warn